ouroboros-consensus/changelog.d/20230224_113043_damian.only_4382_add_getSnapshotFor_benchmarks.md

```diff
@@ -0,0 +1,5 @@
+### Non-Breaking
+
+- Added `mkCapacityBytesOverride`, a convenience function to create an override
+  for the mempool capacity using the provided number bytes.
```

Comment:
Did we decide on present or past tense? I'm mainly using present tense

Reply:
Good point. We have no consistency at the moment. I'll raise this with the team.